src/opentime/rationalTime.h

```diff
@@ -13,61 +13,64 @@ enum IsDropFrameRate : int {
     ForceYes = 1,
 };
     
-    
+constexpr double fabs(double val) noexcept
+{
+    union { double f; uint64_t i; }
+    bits = { val };
+    bits.i &= -1ULL / 2;
```

Comment:
some compilers are going to reject a -1ULL, we should just use `(numeric_limits<uint64_t>::max() /2)` here.

Reply:
Are the parentheses ```()``` needed?

Reply:
that snip was from my crusty brain matter, not documentation :)